src/v2/Apps/Consign/Routes/MarketingLanding/Components/GetPriceEstimate/ConsignPriceEstimateContext.tsx

```diff
@@ -98,21 +146,30 @@ function getActions(dispatch: Dispatch<Action>, relayEnvironment: Environment) {
         }
       )
 
-      actions.setFetching(false)
+      let artistInsight = null
+
+      const edges = response?.priceInsights?.edges
+      if (edges?.length) {
+        artistInsight = edges[0].node
+        const mediums = edges.map(({ node }) => node.medium).sort()
```

Comment:
Sorting this stuff as it comes back, but if we were to leave it alone it would be ordered by demand rank -- maybe something we can do here design-wise to highlight this

Reply:
This means we're also not rendering the most in-demand medium first, right? If not we should reconsider since most users will want what's most in-demand.

Reply:
Cool, i'll remove sort for now and see what folks think